disperser/apiserver/server.go

```diff
@@ -128,8 +130,24 @@ func (s *DispersalServer) DisperseBlobAuthenticated(stream pb.Disperser_Disperse
 		return fmt.Errorf("failed to authenticate blob request: %v", err)
 	}
 
+	// Get the ethereum address associated with the public key. This is just for convenience so we can put addresses instead of public keys in the allowlist.
+	authenticatedAddress := ""
+	// Decode public key
+	publicKeyBytes, err := hexutil.Decode(blob.RequestHeader.AccountID)
+	if err != nil {
+		s.logger.Warn("failed to decode public key (%v): %v", blob.RequestHeader.AccountID, err)
```

Comment:
shouldn't it return error right away? same in L142

Reply:
Sure.